student_analysis/problems.json

```diff
@@ -0,0 +1,78 @@
+,{"_id":{"$oid":"5e80968d99770c00221d371e"},"tags":["math"],"code":"SRM16C","points":300,"name":"Secret Info","description":"Cibin, is the head of an intelligence agency. Since there is a lot of fake information, he wants to share some highly confidential information <b>S</b> to all the defense officials regarding this pandemic.\r\nHere the message to be sent is in a single word consisting of only lowercase English letters. For the confidentiality of the information he decides to encrypt the message. He makes a string <b>M</b> of length <b>N</b>, such that after deleting a substring of <b>non-zero</b> length from <b>M</b>, the remaining string is <b>S</b>. He needs your help to find out the number of all such possible ways he can form the string <b>M</b>.\r\n<b>Note:</b> The string <b>M</b> can be a combination of <b>lowercase</b> English letters.","explainInput":"First line contains the number of test cases <b>T</b>. \r\nFor each test case there is a single line which contains a space separated integer, <b>N</b> and a string <b>S</b>.\r\n","explainOutput":"For each test case, output the number of possible strings modulo <b>10^9+7</b>.","constraints":"1≤ <b>T</b> ≤ 1000\r\n1≤ <b>N</b> ≤10^18\r\n1≤ <b>|S|</b> ≤100\r\n<b>S</b> contains only lowercase English letters.","example":"<b>Input:</b>\r\n3\r\n2 a\r\n3 a\r\n3 ab\r\n<b>Output:</b>\r\n51\r\n1326\r\n76\r\n","explanation":"Following are some examples of 2 lettered words:\r\naa, ab, az, zz, bd, etc.\r\nWe can form <b>26*26 = 676</b> such words but only <b>51</b> out of them contain at least one ‘a’ so that we can form the string <b>S</b>. \r\nEg: We can get <b>‘a’</b> from <b>‘ab’</b> after removing the substring <b>‘b’</b>.\r\n       But we can’t get <b>‘a’</b> from <b>‘cd’</b> or <b>‘tf’</b>.\r\n","contestCode":"SRM16","inputFile":"https://firebasestorage.googleapis.com/v0/b/siesgstarena-1819.appspot.com/o/stdin%2FSRM16%2FeKWMBpauU.in?alt=media","outputFile":"https://firebasestorage.googleapis.com/v0/b/siesgstarena-1819.appspot.com/o/stdout%2FSRM16%2FeKWMBpauU.out?alt=media","createdAt":{"$date":"2020-03-29T12:37:33.055Z"},"updatedAt":{"$date":"2020-03-29T13:50:09.46Z"},"__v":0}
+]
```

Comment:
Dataset files can change over period of time. Add reference in your README from where you have to download dataset files and where you have to keep them for loading..
You may need to make some code changes for that. 

Reference: https://github.com/prabhuomkar/fiducia#dataset